src/components/Collapsible.jsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react"
+
+const Collapsible = ({children, descriptor}) => {
+    const [isHidden, setIsHidden] = useState(true)
+    function hidden () {
+        setIsHidden(!isHidden)
+    }
+
+    return (
+        <div>
+            <button onClick={hidden}> {isHidden? "Show" : "Hide"} {descriptor} </button>
```

Comment:
think about what the user should see when there are no comments - at the moment even when there are no comments, we see a show comments button that doesnt do anything except change to hide comments - this makes the user think the comments have not loaded